pytest_pycharm.py

```diff
@@ -28,5 +28,9 @@ def pytest_exception_interact(node, call, report):
             debugger = pydevd.get_global_debugger()
         pydevd_tracing.SetTrace(None)  # no tracing from here
         debugger.handle_post_mortem_stop(thread, frame, frames_by_id, exception)
```

Comment:
this call is still there, shouldn't we just keep the one in the `try` body?

Reply:
Yes, I did notice that before, but something went wrong with push. Now I have fixed it together with the previous comment.